pkg/transform/testdata/mdox1.yaml

```diff
@@ -8,6 +8,7 @@ gitIgnored: true
 transformations:
   - glob: "README.md"
     path: _index.md
+    popHeader: true
```

Comment:
If popHeader is true on root, why we do this? 🤔

Reply:
Sorry, forgot to remove!

Reply:
It's still there on all examples 🤔